angelleye-includes/angelleye-utility.php

```diff
@@ -2211,8 +2211,17 @@ public static function is_subs_change_payment() {
 	}
 
         public static function angelleye_display_marketing_sidebar($id = null) {
-            wp_enqueue_style('angelleye_marketing_css');
-            require_once( PAYPAL_FOR_WOOCOMMERCE_PLUGIN_DIR . '/template/sidebar.php' );
+            if (false === ( $html = get_transient('angelleye_dynamic_marketing_sidebar_html_pfw') )) {
```

Comment:
Good job here @kcppdevelopers Transient name needs to be different in all of our plugins.